### PR TITLE
Change Slow Start to listen to End events

### PR DESCRIPTION
**Pull request by ascriptmaster**
While I'm here, @Marty-D, does Slow Start display a message if it's active and the ability gets removed?

### CODE REVIEW

**Comment by Marty-D** [comment]
Nope, only if it ends normally after the 5 turns.

**Comment by ascriptmaster** [comment]
Hmmm, I wonder if you could write the volatile's `onEnd` to only display a message if it wears off normally.

**Comment by TheImmortal** [comment]
For Slow Start, you could check if `hasAbility('slowstart')` in the effect's end.

**Comment by Marty-D** [comment]
Alternatively, you can delete the volatile and send a silent `-end`, which TI added support for just recently.

**Comment by TheImmortal** [comment]
Oh yeah. Forgot about that!

File: data/abilities.js [hunk 1]
```diff
@@ -2396,6 +2396,10 @@ exports.BattleAbilities = {
 		onStart: function (pokemon) {
 			pokemon.addVolatile('slowstart');
 		},
+		onEnd: function (pokemon) {
+			delete pokemon.volatiles['slowstart'];
+			this.add('-end', target, 'Slow Start', '[silent]');
```
Comment (TheImmortal) [inline data/abilities.js]: >target

**Comment by ascriptmaster** [comment]
Ok. Speaking of which, I can't verify it right now, but I don't see anything in the Client code right now that would display a message when Slow Start ends. Does it actually display a message when it ends normally?

**Comment by Marty-D** [comment]
Right you are, done in https://github.com/Zarel/Pokemon-Showdown-Client/commit/c4fe4237091f05e4dae9f79d309f65461c15bf1a!

**Pull request merged**